README.md

```diff
@@ -16,6 +22,62 @@ github-issues fetch ustwo/ustwo.com-frontend \
 ```
 
 
+### Upload issues
+
+Create a file `myissues.csv` with the following format:
+
+```csv
+title,body,labels,assignees,milestone_id
```

Comment:
So, the header is not required. The field order is what matters (see new changes).